"How to Write & Deploy an NFT - Part 1, Step 12" doesn't work.

First of, awesome guide, so far it has been real clear. 
https://ethereum.org/en/developers/tutorials/how-to-write-and-deploy-an-nft/

The issue I ran in to, is on step 12 it asks to install ethers.js and this doesn't work as it is instructed.
It says the command is:
npm install --save-dev @nomiclabs/hardhat-ethers 'ethers@^5.0.0'

It gives error in command prompt, 

npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/%27ethers - Not found
npm ERR! 404
npm ERR! 404  ''ethers@5.0.0'' is not in the npm registry.
npm ERR! 404 Your package name is not valid, because
npm ERR! 404  1. name can no longer contain special characters ("~'!()*")
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

this is also the command hardhat themselves have in their tutorial on:
https://hardhat.org/plugins/nomiclabs-hardhat-ethers.html

though on the link you supplied to the ethers.js (which is the same one hardhat links to)
https://github.com/ethers-io/ethers.js/

says the current command to install them is:
/home/ricmoo/some_project> npm install --save ethers

which does work for me so i used that to install and moved on.

My problem now is that during step 16 where you have to compile your contract, it says "ReferenceError: ethers is not defined".
I feel the issue is with not being able to follow step 12 exactly.


Doing this on Windows 10, in command window.

I may be an idiot and doing something other silly... I am doing a beginner tutorial after all ^^, but I do feel step 12 doesn't work anymore, while all others steps went super smooth :) 

Stay Awesome :D

--- a/src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md
+++ b/src/content/developers/tutorials/how-to-write-and-deploy-an-nft/index.md
@@ -244,7 +244,7 @@ Hardhat makes it super easy to integrate [Plugins](https://hardhat.org/plugins/)
 
 In your project directory type:
 
-    npm install --save-dev @nomiclabs/hardhat-ethers 'ethers@^5.0.0'
+    npm install --save-dev @nomiclabs/hardhat-ethers ethers@^5.0.0
 
 We’ll also require ethers in our hardhat.config.js in the next step.
 

